concrete/bootstrap/configure.php

```diff
@@ -326,6 +326,11 @@
  */
 const USER_FOREVER_COOKIE_LIFETIME = 1209600; // 14 days
 const USER_CHANGE_PASSWORD_URL_LIFETIME = 7200;
+/**
+ * Use Config::get('concrete.misc.live_version_status_on_scheduled_version_approval')
+ */
+const APPROVED = 'approved';
```

Comment:
I don't think we want this here. I think you can just use APPROVED and UNAPPROVED in the files directly.